packages/x-date-pickers/src/AdapterDayjs/AdapterDayjs.test.tsx

```diff
@@ -0,0 +1,138 @@
+          render(<DateTimePicker />);
+
+          expectInputValue(
+            screen.getByRole('textbox'),
+            localizedTexts[localeKey].placeholder,
+            true,
+          );
+        });
+
+        it('should have well formatted value', () => {
+          render(<DateTimePicker value={adapter.date(testDate)} />);
+
+          expectInputValue(screen.getByRole('textbox'), localizedTexts[localeKey].value, true);
+        });
+      });
+    });
+  });
+
+  describe('UTC plugin', () => {
+    const { render } = createPickerRenderer({
+      clock: 'fake',
+      adapterName: 'dayjs',
+    });
+
+    it('should not create UTC dates when no instance passed', () => {
+      const onChange = spy();
+      render(<DateCalendar defaultValue={dayjs('2022-04-10')} onChange={onChange} />);
+
+      userEvent.mousePress(screen.getByRole('gridcell', { name: '1' }));
+
+      const date = onChange.lastCall.firstArg;
+      expect(date).to.not.equal(null);
+      expect(date!.isUTC()).to.equal(false);
+    });
+
+    it('should create UTC dates when the instance is a UTC instance', () => {
+      const onChange = spy();
+      render(
+        <LocalizationProvider dateAdapter={AdapterDayjs} dateLibInstance={dayjs.utc}>
+          <DateCalendar defaultValue={dayjs.utc('2022-04-10')} onChange={onChange} />
+        </LocalizationProvider>,
+      );
+
+      userEvent.mousePress(screen.getByRole('gridcell', { name: '1' }));
+
+      const date = onChange.lastCall.firstArg;
+      expect(date).to.not.equal(null);
+      expect(date!.isUTC()).to.equal(true);
+    });
+
+    it('should not loose UTC when dragging', function test() {
+      if (!document.elementFromPoint) {
+        this.skip();
+      }
+
+      const dataTransfer = new MockedDataTransfer();
+      const { executeDateDrag } = buildPickerDragInteractions(() => dataTransfer);
+
+      const onChange = spy();
+      const initialValue: [any, any] = [dayjs.utc('2022-04-10'), dayjs.utc('2022-04-12')];
+      render(
+        <LocalizationProvider dateAdapter={AdapterDayjs} dateLibInstance={dayjs.utc}>
+          <DateRangeCalendar onChange={onChange} defaultValue={initialValue} calendars={1} />
+        </LocalizationProvider>,
+      );
+
+      executeDateDrag(
+        screen.getByRole('gridcell', { name: '12', selected: true }),
+        screen.getByRole('gridcell', { name: '13' }),
+        screen.getByRole('gridcell', { name: '14' }),
+      );
+
+      expect(onChange.callCount).to.equal(1);
+      const [startDate, endDate] = onChange.lastCall.firstArg;
+      expect(startDate.isUTC()).to.equal(true);
+      expect(endDate.isUTC()).to.equal(true);
+    });
+  });
+
+  it('should return the correct week number', () => {
+    const localizedAdapter = new AdapterDayjs({ locale: 'fr' });
+
+    const dateToTest = localizedAdapter.date(new Date(2022, 10, 10));
+
+    expect(localizedAdapter.getWeekNumber!(dateToTest)).to.equal(45);
+  });
```

Comment:
Maybe put this test inside of `Localization` describe block? After all, it is taken from the original test file 🤔